c2corg_ui/static/partials/cards/areas.html

```diff
@@ -4,7 +4,7 @@
   <br>
   <div class="list-item-info flex baseline-align">
     <div class="quality" ng-if="doc['quality']" ng-class="doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{doc['quality'] | translate}}" tooltip-placement="left"></div>
-    <span x-translate>{{doc['area_type']}}</span>
+         uib-tooltip="{{'quality' | translate}} : {{doc['quality'] | translate}}" tooltip-placement="left"></div>
+    {{doc['area_type'] | translate}}
```

Comment:
No, because the extractor is smart enough to understand ``doc['area_type']`` is a variable, not a fixed string.

I have tried to run the extractor on this file like this:
```
node tools/extract-messages.js c2corg_ui/static/partials/cards/areas.html > foo.txt
```
and only ``quality`` is extracted. Which is correct.